x-pack/plugins/cases/common/constants/index.ts

```diff
@@ -54,6 +54,7 @@ export const CASE_PUSH_URL = `${CASE_DETAILS_URL}/connector/{connector_id}/_push
 export const CASE_REPORTERS_URL = `${CASES_URL}/reporters` as const;
 export const CASE_STATUS_URL = `${CASES_URL}/status` as const;
 export const CASE_TAGS_URL = `${CASES_URL}/tags` as const;
+export const CASE_CATEGORIES_URL = `${CASES_URL}/categories` as const;
```

Comment:
Let's make the endpoint internal for starters. No need to support it from the beginning.